scene/main/scene_tree.h

```diff
@@ -64,9 +65,12 @@ class SceneTreeTimer : public Reference {
 	void set_pause_mode_process(bool p_pause_mode_process);
 	bool is_pause_mode_process();
 
+	void set_ignore_time_scale(bool p_ignore);
+	bool is_ignore_time_scale();
+
 	void release_connections();
 
-	SceneTreeTimer();
+	SceneTreeTimer(){};
```

Comment:
```suggestion
	SceneTreeTimer() {}
```

Reply:
But the space was removed by clang-format...

Reply:
Yes, but that's because the trailing semicolon is a bug.